meson.build

```diff
@@ -2801,6 +2814,7 @@ status = [
         'symbolic gateway hostnames:        @0@'.format(', '.join(gateway_hostnames)),
 
         'default DNSSEC mode:               @0@'.format(default_dnssec),
+        'default DNS-over-TLS mode:         @0@'.format(default_dnstls),
```

Comment:
i figure using gnutls for this is fine for now. I have the suspicion now that everybody appears to settle on OpenSSL these days, and we should gradually move over to that.

Reply:
i think using gnutls is fine here, fwiw.  knot-resolver (one of the popular dns resolvers that implements dns-over-tls) uses gnutls and has no problem with it.